packages/spanner/test/transaction.js

```diff
@@ -180,6 +180,7 @@ describe('Transaction', function() {
 
   describe('begin', function() {
     var OPTIONS = {
+      readOnly: true,
```

Comment:
below in EXPECTED_REQ_OPTS you set readOnly to OPTIONS, so it will look like
```
readOnly: {
   readOnly: true,
   boundOptions: true,
   returnReadTimestamp: true,
}
```

So is that expected?

Reply:
This test is a little confusing, but essentially yes. We have an abstract class of sorts that will format the timestamp options and in the test we stub that class.

https://github.com/callmehiphop/gcloud-node/blob/84befe838b8923d1a61562c94f251e0afb42cec7/packages/spanner/test/transaction.js#L47

So we're essentially just testing that the value of `transaction.options` is sent along in the request.